src/Extensions/SiteTreeSubsites.php

```diff
@@ -484,14 +484,7 @@ public function augmentSyncLinkTracking()
 
                         $origDisableSubsiteFilter = Subsite::$disable_subsite_filter;
                         Subsite::disable_subsite_filter(true);
-                        $candidatePage = DataObject::get_one(
-                            SiteTree::class,
-                            "\"URLSegment\" = '"
-                            . Convert::raw2sql(urldecode($rest))
-                            . "' AND \"SubsiteID\" = "
-                            . $subsiteID,
-                            false
-                        );
+                        $candidatePage = SiteTree::get()->filter(['URLSegment' => urldecode($rest), 'SubsiteID' => $subsiteID])->first();
```

Comment:
This line needs to be split in order to make the linter happy :)
I suggest making the array multi-line.

Reply:
Fixed now